assets/src/domain/eventEditor/ui/tickets/ticketsList/tableView/ticketsListTableRow.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
@tn3rb should `TicketRegistrationsLink` also be added, as in datetime case ?

Reply:
yes... there's another issue for fixing the entity details panels for both dates and tickets.
so don't worry about that here